pandas/formats/style.py

```diff
@@ -851,39 +851,180 @@ def set_properties(self, subset=None, **kwargs):
         return self.applymap(f, subset=subset)
 
     @staticmethod
-    def _bar(s, color, width):
-        normed = width * (s - s.min()) / (s.max() - s.min())
+    def _bar_left(s, color, width, base):
+        """
+        The minimum value is aligned at the left of the cell
+        .. versionadded:: 0.17.1
```

Comment:
@jreback told me in an earlier review that I didn't need the versionadded tags for these, so I delete them all. Which?